pkgs/applications/graphics/foxotron/default.nix

```
@@ -35,6 +35,10 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-s1eWZMVitVSP7nJJ5wXvnV8uI6yto7LmvlvocOwVAxw=";
   };
 
+  patches = [
+    ./switch-FBX-to-cpp11-stdint.patch
```

Instead of vendoring a patch, we should fetch it from assimp. Darwin seems to require at least one more patch as well.

```suggestion
    (fetchpatch {
      name = "0001-assimp-Include-cstdint-for-std-uint32_t.patch";
      url = "https://github.com/assimp/assimp/commit/108e3192a201635e49e99a91ff2044e1851a2953.patch";
      stripLen = 1;
      extraPrefix = "externals/assimp/";
      hash = "sha256-rk0EFmgeZVwvx3NJOOob5Jwj9/J+eOtuAzfwp88o+J4=";
    })
    (fetchpatch {
      name = "0002-assimp-Disable-diagnostic-for-LogStream-comparator.patch";
      url = "https://github.com/assimp/assimp/commit/58cae73401602a7ea8aaf61682a6215b8cdd25c3.patch";
      stripLen = 1;
      extraPrefix = "externals/assimp/";
      hash = "sha256-/jWv2G116wUaMi+F0SzRk1wgKu9nfJsWYTXYS2U76wE=";
    })
```

---

@OPNA2608 Thank you for digging those up.
I've rebased your suggestion into my original commit and force-pushed the result. I hope that's the correct way to apply this patch?